CHANGES.md

```diff
@@ -117,12 +151,67 @@ description indicates whether it is SOURCE-COMPATIBLE, BINARY-COMPATIBLE, or BRE
-* Rename libsel4 config option ENABLE_SMP_SUPPORT to CONFIG_ENABLE_SMP_SUPPORT to be namespace compliant.
+* Correct check for message length in `SchedControl_ConfigureFlags`
+* Allow lazy SchedContext rebind. Before, binding a scheduling context to a TCB was not allowed if the SC was bound to a
+  notification object. Also, binding an SC to a notification was not allowed if that scheduling context was already
+  bound to a TCB. Without these restriction it is much easier to move scheduling contexts around: In effect having a SC
+  bound on both the TCB and a notification acts as if the thread is running on a donated SC which will be returned when
+  the tasks calls `Recv`/`Wait`, which is done by `maybeReturnSchedContext()`.
+* Fix scheduling context use-after-free on SMP
+* Only charge budgets for non-idle thread SCs
+* ARM+MCS: Introduce `TIMER_OVERHEAD_TICKS`. For ARM currently `TIMER_PRECISION` exists, but that is in microseconds and
+  not fine-grained enough. `TIMER_OVERHEAD_TICKS` is needed to make periodic tasks synchronous with the system clock. If
+  this value is non-zero every period will be extended with the overhead of taking an interrupt and reading the system
+  clock. To avoid this drift, the configured value should be set to at least the average overhead.
+* SMP: Do not use cross-node `ksCurTime` assuming they are in sync (which they are not), instead use
+  `NODE_STATE(ksCurTime)`.
+* SMP: Add clock synchronisation test on boot
+
+#### Other Changes
+
+* Fix: Don't clobber msgInfo register in `PageGetAddress`, `ASIDControlInvocation`, `ARMCBInvocation`,
+  `A32PageDirectoryGetStatusBits`, `X86PortIn`, `WriteVMCS`, `ReadVMCS`, `ConfigureSingleStepping`, and `GetBreakpoint`.
+* `libsel4`: Make `bootinfo` consistent. Some slot positions in the rootnode would depend on configuration. However that
+  makes it difficult to add new root caps, especially if multiple caps only exist based on configuration. Make all caps
+  always there, but null if not configured.
+* Improved consistency and completeness of user manual
+* Added manual for bitfield generator
+* `bitfield_gen`: allow non-contiguous tag fields. A tagged union can now optionally use multiple fields to indicate the
+  tag. These are called "sliced" tags in the code. The tag fields have to be at the same position and width in each
+  block of the tagged unions, and all tag fields have to be within the same word. See the manual for details.
+* Make `CONFIG_PRINTING` and `CONFIG_DEBUG_BUILD` usable independently from each other
+* Overall debug printing improvements
+* Rename libsel4 config option `ENABLE_SMP_SUPPORT` to `CONFIG_ENABLE_SMP_SUPPORT` to be namespace compliant.
 * Removed obsolete define `HAVE_AUTOCONF`
+* Remove userError from `seL4_ReplyRecv` path, because it too often incorrectly warns about legitimate operations
+* Update GDB macros
+* Add support for `cmake install <dir>` for final build and config artefacts
```

Comment:
I think it should be `cmake --install <dir`> instead?

Reply:
yes, good catch